adsingestp/parsers/base.py

```diff
@@ -471,6 +471,35 @@ class BaseBeautifulSoupParser(IngestBase):
     out of the input XML stream.
     """
 
+    fix_ampersand = re.compile(r"(&amp;)(.*?)(;)")
+
```

Comment:
I don't know if you have ever encountered this in this context: I have seen cases where ampersands got encoded as `__amp__amp;`

Reply:
Do you remember which publisher(s) specifically?  I'm looking for an example to make a unit test with.

Reply:
This file has an example: `/proj/ads/references/sources/MNRAS/0423/iss4.wiley2.xml`